operator/pkg/apis/cassandra/v1beta1/webhook.go

```diff
@@ -45,6 +44,12 @@ func ValidateSingleDatacenter(dc CassandraDatacenter) error {
 		return err
 	}
 
+	if dc.Spec.ServerType == "cassandra" && dc.Spec.DseWorkloads != nil {
+		if dc.Spec.DseWorkloads.AnalyticsEnabled || dc.Spec.DseWorkloads.GraphEnabled || dc.Spec.DseWorkloads.SearchEnabled {
+			return attemptedTo("cannot enable DSE workloads if server type is Cassandra")
```

Comment:
```suggestion
			return attemptedTo("enable DSE workloads when server type is Cassandra")
```

the error message will be "attempted to ...", so this is just a grammar fixup